crates/router/src/connector/riskified.rs

```diff
@@ -173,7 +184,17 @@ impl
         req: &frm_types::FrmCheckoutRouterData,
         _connectors: &settings::Connectors,
     ) -> CustomResult<RequestContent, errors::ConnectorError> {
-        let req_obj = riskified::RiskifiedPaymentsCheckoutRequest::try_from(req)?;
+        let amount = convert_amount(
+            self.amount_converter,
+            MinorUnit::new(req.request.amount),
+            req.request
+                .currency
+                .ok_or(errors::ConnectorError::MissingRequiredField {
+                    field_name: "currency",
+                })?,
+        )?;
```

Comment:
Similar change needs to be done wherever amount is being used like `FrmTransactionRouterData` .